### PR TITLE
ref/update_ios_nativead_to_send_bool

### PR DESCRIPTION
**Pull request by alhiwatan**
Update the iOS nativeAd to send boolean values instead of integers to simplify the Flutter side.

### CODE REVIEW

File: applovin_max/ios/Classes/AppLovinMAXNativeAdView.m [hunk 1]
```diff
@@ -461,9 +461,9 @@ - (void)sendAdLoadedReactNativeEventForAd:(MANativeAd *)ad
         nativeAdInfo[@"mediaContentAspectRatio"] = @(ad.mediaContentAspectRatio);
     }
     
-    nativeAdInfo[@"isIconImageAvailable"] = @(ad.icon != nil);
-    nativeAdInfo[@"isOptionsViewAvailable"] = @(ad.optionsView != nil);
-    nativeAdInfo[@"isMediaViewAvailable"] = @(ad.mediaView != nil);
+    nativeAdInfo[@"isIconImageAvailable"] = (ad.icon != nil) ? @YES : @NO;
```
Comment (thomasmso) [inline applovin_max/ios/Classes/AppLovinMAXNativeAdView.m:464]: I'd prefer less code: `@(ad.icon != nil)`

**Review by thomasmso — APPROVED** [review]

File: applovin_max/ios/Classes/AppLovinMAXNativeAdView.m (hunk 1 above)
Comment (alhiwatan) [reply applovin_max/ios/Classes/AppLovinMAXNativeAdView.m:464]: I do, but that will be translated into an integer, 1 or 0, on the Flutter side.

Comment (thomasmso) [reply applovin_max/ios/Classes/AppLovinMAXNativeAdView.m:464]: I see

Comment (thomasmso) [reply applovin_max/ios/Classes/AppLovinMAXNativeAdView.m:464]: LG then.

**Pull request merged**